tools/api/ontology.py

```diff
@@ -0,0 +1,58 @@
+import sys
+from typing import List
+
+sys.path.insert(1, '../validators/ontology_validator/')
+
+from yamlformat.validator.entity_type_lib import EntityType
+from yamlformat.validator.presubmit_validate_types_lib import ConfigUniverse
+from yamlformat import constants
+
+from model import StandardField
+from model import EntityTypeField
+
+class Ontology(object):
+    """Class defining an implementation of Google's ConfigUniverse as an ontology
+
+       Args:
+            universe: an instantiated ConfigUniverse Object
+
+       Attributes:
+            universe: A ConfigUniverse object detailing the various universes in the ontology
+
+       Returns:
+            An instance of Ontology class
+    """
+
+    def __init__(self, universe: ConfigUniverse):
+        super().__init__()
+        self.universe = universe
+
+    def GetFieldsForTypeName(self, namespace: str, entity_type_name: str, required_only: bool = False) -> List[EntityTypeField]:
+        """gets a list of fields for a given typename within a namespace
```

Comment:
Be sure to check the automated linter on Github - this would fail because the line doesn't start with a capital letter and doesn't end with a '.'.  You'll have to pass the linter to merge.